pootle/core/views.py

```diff
@@ -778,7 +778,7 @@ def get_context_data(self, *args, **kwargs):
             raise Http404(
                 ValidationError(search_form.errors).messages)
 
-        total, start, end, units_qs = search_backend.get(Unit)(
+        total, __, __, units_qs = search_backend.get(Unit)(
```

Comment:
In cases where several `__` are assigned at once I would keep the `whatever_` syntax.